apps/server/src/modules/files-storage/mapper/file-response.builder.ts

```diff
@@ -0,0 +1,9 @@
+import { IGetFile, IGetFileResponse } from '../interface';
+
+export class FileResponseBuilder {
```

Comment:
Why is this called "...Builder" while FilesStorageMapper is called "...Mapper"? They do the same sort of thing, don't they? 

Open question. I don't have any suggestion or so. It just struck me when I read the names.